mflowgen/full_chip/construct.py

```diff
@@ -206,7 +201,7 @@ def construct():
   synth.extend_inputs( read_design.all_outputs() )
   synth.extend_inputs( ["cons_scripts"] )
 
-  power.extend_outputs( ["design.gds.gz"] )
+  power.extend_outputs( ["design-merged.gds"] )
```

Comment:
Did you make a change somewhere to produce this output?

Reply:
yes, see https://github.com/StanfordAHA/garnet/pull/666/commits/15dca4b018fc137b435f0f39468f15b7b304c9a9